Assets/Scripts/Controller Based Shadow Detector/Drawer/DrawedShadowObject.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DrawedShadowObject : ShadowObject
```

Comment:
ControllerBased의 기능을 수행하는 ShadowObject니까 Drawed보다는 ControllerBased라는 말이 좀 더 명확할 것 같습니다.
```suggestion
public class ControllerBasedShadowObject : ShadowObject
```